schema/example.go

```diff
@@ -0,0 +1,173 @@
+	}
+
+	volunteer := volunteerSchema{
+		Organization: "Xyz",
+		Position:     "Open Source Contributor",
+		Location:     location,
+		URL:          "http://xyz.example.com",
+		StartDate:    "2014-06-29",
+		EndDate:      "2017-06-29",
+		Summary:      "Frontend developer",
+		Highlights:   []string{"Invited as a speaker in Xyzcon'17"},
+	}
+
+	publication := publicationSchema{
+		Name:        "Deep learning and Artificial Intelligence",
+		Publisher:   "XYZ, Computer Magazine",
+		ReleaseDate: "2015-08-01",
+		Resources:   []resourceSchema{resource},
+		URL:         "http://www.computer.org.example.com/csdl/mags/co/2015/10/rx069-abs.html;format:uri",
+		Summary:     "Discussion of the advent of deep learning and artificial intelligence",
+	}
+
+	legal := legalSchema{
+		Name:            "XYZ's patent on LZW compression, a fundamental part of the widely used GIF graphics format",
+		LegalType:       "Patent, Trademark, Copyright",
+		Description:     "Some legal document!",
+		ApplicationDate: "2015-08-01",
+		GrantDate:       "2016-09-01",
+		EndDate:         "2020-09-3",
+		Resources:       []resourceSchema{resource},
+		IDNumber:        "JP2004369746A",
+	}
+
+	skill := skillSchema{
+		Name: "Web Development",
+		Keyword: []struct {
+			Name  string      `jsonschema:"name;description:Add the skill name"`
+			Score scoreSchema `jsonschema:"score;additionalProperties;description:Score for the skill name"`
+		}{
+			{
+				Name:  "HTML",
+				Score: score,
+			},
+		},
+	}
+
+	award := awardSchema{
+		Title:   "Awarded Software Process Achievement Award",
+		Date:    "2016-06-12",
+		Awarder: "IEEE",
+		Summary: "Received for my work in Deep learning and AI",
+	}
+
+	project := projectSchema{
+		Name:        "File Transfer application",
+		Location:    location,
+		Description: "Developed a client and server based application",
+		Highlights:  []string{"used Java AWT and Swing for client side userinterface"},
+		Keywords:    []string{"Java"},
+		StartDate:   "2016-06-29",
+		EndDate:     "2017-03-02",
+		Resources:   []resourceSchema{resource},
+		URL:         "http://www.example.org/csdl/mags/co/1996/10/rx069-abs.html",
+		Roles:       []string{"Team Lead", "Speaker", "Writer"},
+		Entity:      "greenpeace",
+		Type:        "volunteering",
+	}
+
+	certificate := certificateSchema{
+		Code:          "1Z0-062",
+		Name:          "XYZ Certified Application Specialist (MCAS)",
+		Website:       "http://www.example.org",
+		Verification:  "http://www.example.org",
+		GrantDate:     "2017-06-29",
+		Score:         score,
+		EndDate:       "2017-06-29",
+		DoesNotExpire: true,
+	}
+
+	reference := referenceSchema{
+		Name:      "Stephan Mark",
+		Company:   "Xyz",
+		Position:  "Senior Software Engineer",
+		Reference: "Joe blogs was a great employee, who turned up to work at least once a week. He exceeded my expectations when it came to doing nothing.",
+	}
+
+	language := languageSchema{
+		Language: "English",
+		Score:    score,
```

Comment:
Not perfectly sure about the usage of score here. Language is usually described in "fluent", "native" string type attributes. But this could probably be reflected with our scoreSchema, but we should at least add a more realistic language example here instead of just the default GPA, which doesn't make complete sense.

Reply:
Fixed in 570748d476f068d3be54b49ed54ae44398a2eeaa